toolkit/tools/internal/pkgjson/pkgjson_test.go

```diff
@@ -887,3 +1005,80 @@ func TestShouldFailToConvertPackageListEntryWithWhitespacesInVersion(t *testing.
 
 	assert.Error(t, err)
 }
+
+func TestSimpleSort(t *testing.T) {
+	pkgRepo := &PackageRepo{
+		Repo: []*Package{
+			{
+				Provides: &PackageVer{Name: "pkgB", Version: "1.0.0"},
+			},
+			{
+				Provides: &PackageVer{Name: "pkgA", Version: "1.0.0"},
+			},
+		},
+	}
+
+	SortPackageList(pkgRepo.Repo)
+	assert.Equal(t, "pkgA", pkgRepo.Repo[0].Provides.Name)
+	assert.Equal(t, "pkgB", pkgRepo.Repo[1].Provides.Name)
+
+}
```

Comment:
Do we need `PackageRepo`?
```suggestion
	pkgRepo := []*Package{
		{
			Provides: &PackageVer{Name: "pkgB", Version: "1.0.0"},
		},
		{
			Provides: &PackageVer{Name: "pkgA", Version: "1.0.0"},
		},
	}

	SortPackageList(pkgRepo)
	assert.Equal(t, "pkgA", pkgRepo[0].Provides.Name)
	assert.Equal(t, "pkgB", pkgRepo[1].Provides.Name)
}
```

I think we're also not using `PackageRepo` below.

Reply:
Fixed. This used to be a member function of the repo but I realized that wasn't useful in the grapher but didn't update the tests fully.